tls/s2n_resume.h

```diff
@@ -70,3 +70,8 @@ typedef enum {
 extern int s2n_allowed_to_cache_connection(struct s2n_connection *conn);
 extern int s2n_resume_from_cache(struct s2n_connection *conn);
 extern int s2n_store_to_cache(struct s2n_connection *conn);
+
+/* This function will be labeled S2N_API and become a publicly visible api once we release the psk API. */
+int s2n_config_set_requested_new_ticket_count(struct s2n_config *config, uint8_t num);
+/* This function will be labeled S2N_API and become a publicly visible api once we release the psk API. */
+int s2n_connection_request_new_tickets(struct s2n_connection *conn, uint8_t num);
```

Comment:
There was some discussion previously about whether these are the right names.
* `s2n_config_set_requested_new_ticket_count` -> `s2n_config_set_initial_new_ticket_count`?
* `s2n_connection_request_new_tickets` kind of sounds like the API will immediately send some message / request. Maybe `s2n_connection_add_new_tickets`? `s2n_connection_add_new_tickets_to_send`?

Reply:
I find these names confusing as well, we could simplify it to clearly state its intended purpose. 
`s2n_config_set_initial_tickets_to_send_count` seems more close to what the first API does and `s2n_connection_add_new_tickets_to_send` for the second API.

Reply:
```s2n_config_set_initial_ticket_count``` and ```s2n_connection_add_new_tickets_to_send```?